source/ftrack_connect_pipeline_qt/ui/assembler/base/__init__.py

```diff
@@ -453,10 +453,10 @@ def extract_components(self, versions):
 class AssemblerListBaseWidget(AssetListWidget):
     '''Base for asset lists within the assembler'''
 
-    def __init__(self, assembler_widget, parent=None):
+    def __init__(self, assembler_widget, component_widget_class, parent=None):
         self._assembler_widget = assembler_widget
         super(AssemblerListBaseWidget, self).__init__(
-            self._assembler_widget.model, parent=parent
+            self._assembler_widget.model, component_widget_class, parent=parent
```

Comment:
Why are we passing a class to the AssemblerListBaseWidget instead of initializing this class inside the AssemblerListBaseWidget?

Reply:
See comment above